examples/README.md

```diff
@@ -0,0 +1,93 @@
+but should be taken as just an example.
+
+**[extensions-collection/collection.json](extensions-collection/collection.json)** contains a small number of items, that demonstrate
+more functionality available in STAC [extensions](../extensions/). These are linked to directly from the individual extensions. These
+items follow the recommendations for [Catalog Layout Best Practices](../best-practices.md#catalog-layout).
+
+## In Depth
+
+As mentioned above, the files in this examples directory form valid STAC implementations. They are all based on a 
+fictional remote sensing company called 'Remote Data', with a URL at remotedata.io. This domain has not been set up, so those links
+will not work, but any valid data provider should provide valid links to their homepage. 
+
+The examples use the `rd:` prefix to show how providers can use custom fields when there are not set fields. In the examples these
+do not link to a schema which is completely valid, but it is recommended that providers do write a JSON schema that can validate 
+their custom fields (we will work to add a schema in the future). 
+
+### Differences with STAC Best Practices
+
+One of the most important documents in this repository is the one about [best practices](../best-practices.md). It describes a number
+of practical recommendations gained by people actually implementing STAC. The core spec is designed to be as flexible as possible, so
+that it is not too rigid and unable to handle unanticipated needs. But we recommend following as many of the best practices as is 
+feasible, as it will help ensure various STAC tools work much better. The examples in this folder don't align with all the best
+practices, mostly because they are meant to demonstrate things as tersely as possible, and also because they live directly inside
+a github repository. As many people will look at these examples and take them as 'how things should be' we felt its important to
+highlight where things here differ from the actual best practices.
+
+#### Relative vs Absolute Links
+
+One of the [main recommendations](../best-practices.md#use-of-links) of the best practices document is to aim for consistency in 
+the use of relative versus absolute links. STAC implementations should use relative and absolute links as recommended by one of
+the 'catalog types': '[self-contained](../best-practices.md#self-contained-catalogs)', or '[published](../best-practices.md#published-catalogs)' - absolute or relative. 
+
+The catalogs contained here don't follow any of the recommendations in full. They are closest to a 'relative published catalog',
+as they use a self link at the root and then mostly have relative links. This enables them to be mostly portable, while also having
+a stable online location that can be referenced. But they use absolute links to their assets, due to the fact that they are stored in 
```

Comment:
Talking about this is a bit confusing. This sounds like absolute assets is something not so good although it's totally valid, e.g. if a catalog links to s3 buckets from a HTTP catalog why is that not a best practice? Couldn't we make one of the best practices work in GH? We can get predictable URLs from tags...

Reply:
Oh yeah, we talked this - decided we should have a new category of 'self-contained catalog' that includes absolute url's. Just like we have 'relative' and 'absolute' published catalogs we can have a distinction for self-contained ones, if the assets are relative or absolute.